llvm/docs/Remarks.rst

```diff
@@ -57,6 +57,11 @@ Enabling optimization remarks
 There are two modes that are supported for enabling optimization remarks in
 LLVM: through remark diagnostics, or through serialized remarks.
 
+See also the clang flags
+`-Rpass <https://clang.llvm.org/docs/UsersManual.html#options-to-emit-optimization-reports>`_
```

Comment:
Is it possible to directly reference the sections via RST, so we don't hard-code them? This would avoid potentially having them broken in the future.

If not, that should be fine.

Reply:
(The problem is that this is in the LLVM RST, while the target is in the clang. They are hosted on separate subdomains, so I am not sure how that would work)